metrics/prometheus_machine_test.go

```diff
@@ -31,7 +31,8 @@ const machineMetricsFile = "testdata/prometheus_machine_metrics"
 const machineMetricsFailureFile = "testdata/prometheus_machine_metrics_failure"
 
 func TestPrometheusMachineCollector(t *testing.T) {
-	collector := NewPrometheusMachineCollector(testSubcontainersInfoProvider{}, container.AllMetrics)
+	denyList, _ := NewDenyList([]string{})
```

Comment:
Do not ignore errors, please. Assert that no error was returned from `NewDenyList()`.

Reply:
ok